vunit/vivado/vivado.py

```diff
@@ -13,11 +13,12 @@
 from pathlib import Path
 
 
-def add_from_compile_order_file(vunit_obj, compile_order_file, dependency_scan_defaultlib=True):
+def add_from_compile_order_file(vunit_obj, compile_order_file, dependency_scan_defaultlib=True,
+                                fail_on_non_hdl_files=True):
```

Comment:
```suggestion
                                fail_on_non_hdl_files=True):  # pylint: disable=too-many-locals
```
Should fix the pylint locals warning.

(I guess that in general, the fewer commits the better.)